src/ucp/wireup/wireup.c

```diff
@@ -30,6 +30,48 @@ static size_t ucp_wireup_msg_pack(void *dest, void *arg)
     return sizeof(ucp_wireup_msg_t) + req->send.length;
 }
 
+static const char* ucp_wireup_msg_str(uint8_t msg_type)
```

Comment:
q: what do you think to do something like (e.g. to not forget to update this function if new `msg_type` is added):

usc/sys/preprocessor.h:
```
#define UCS_PP_VAL(_val) _val
```

wireup.h:
```
#define UCP_WIREUP_MSG(FUNC) \
    FUNC(UCP_WIREUP_MSG_PRE_REQUEST),\
    FUNC(UCP_WIREUP_MSG_REQUEST), \
    FUNC(UCP_WIREUP_MSG_REPLY), \
    FUNC(UCP_WIREUP_MSG_ACK)

enum {
    UCP_WIREUP_MSG(UCS_PP_VAL)
};
```
wireup.c
```
static const char *ucp_wireup_msg_str[] = {
    UCP_WIREUP_MSG(UCS_PP_MAKE_STRING);
};
```

or is it better to avoid macro here too?

Reply:
i think it will be somewhat more complicated, and there is no need for this..